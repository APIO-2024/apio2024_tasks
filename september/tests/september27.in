txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 1 1 3 4 4 6 6 7
5 8 7 9 6 4 3 2 1
10 1
0 1 2 3 4 4 5 7 7
8 6 9 7 4 5 3 2 1
10 1
0 1 2 3 3 4 6 7 7
5 9 7 6 4 3 8 1 2
10 1
0 1 1 3 4 5 5 7 7
2 8 6 9 5 7 4 3 1
10 1
0 0 2 2 3 5 6 6 8
4 1 7 9 8 6 5 3 2
10 1
0 1 1 3 4 5 6 6 8
7 9 2 8 6 5 3 4 1
10 1
0 0 1 3 3 4 6 7 8
5 2 9 7 6 8 4 3 1
10 1
0 1 2 2 3 5 5 6 8
7 4 9 8 6 5 3 2 1
10 1
0 0 2 3 3 5 5 7 8
1 6 4 8 9 7 5 2 3
10 1
0 1 1 2 4 4 5 7 8
9 8 7 5 6 4 3 2 1
10 1
0 0 1 3 4 5 6 6 7
9 8 2 6 7 4 3 1 5
10 1
0 1 1 3 4 5 6 6 8
9 2 7 8 6 5 4 3 1
10 1
0 1 1 3 4 5 5 7 8
2 6 8 7 9 4 3 5 1
10 1
0 1 2 2 4 4 6 7 7
8 7 9 5 6 3 2 4 1
10 1
0 0 2 3 4 4 5 7 8
6 1 9 7 8 5 4 3 2
10 1
0 1 1 2 3 4 6 7 7
8 9 7 6 5 4 2 3 1
10 1
0 0 2 2 3 4 5 7 7
6 1 8 4 9 7 5 3 2
10 1
0 1 1 3 4 5 6 7 7
2 9 8 7 6 5 4 3 1
10 1
0 0 2 3 4 5 5 6 7
1 6 8 9 7 5 4 2 3
10 1
0 1 1 2 4 5 6 7 7
9 8 7 3 6 5 4 2 1
40 1
0 1 0 3 2 3 4 7 7 9 9 9 12 12 13 15 15 17 18 17 19 21 20 21 24 24 26 27 26 28 29 30 30 31 34 33 36 36 37
8 38 23 16 22 5 35 6 10 39 14 32 37 36 11 25 34 20 33 2 30 31 1 28 27 29 26 24 21 19 18 15 17 13 12 9 4 7 3
40 1
0 0 1 1 3 3 5 7 8 9 10 9 12 11 12 13 14 16 17 17 19 20 20 22 24 23 26 26 27 29 28 30 32 32 34 35 35 37 38
25 31 39 33 38 6 37 18 2 4 28 21 24 19 15 36 22 16 35 13 12 34 32 30 29 27 26 23 20 17 14 11 10 8 9 7 5 3 1
40 1
0 0 2 1 4 3 4 6 8 9 9 9 12 13 13 15 14 15 18 17 20 19 20 21 22 23 24 27 26 27 30 30 32 32 33 34 35 35 37
5 29 16 10 39 36 34 31 37 38 26 22 25 23 11 19 28 18 15 35 33 32 30 7 27 1 4 24 21 17 20 14 13 12 9 8 6 3 2
40 1
0 0 1 1 2 3 4 5 8 8 8 11 11 11 14 15 15 15 17 19 18 19 22 22 22 24 26 25 28 28 30 30 30 33 34 35 35 35 37
16 21 9 10 36 32 7 23 18 39 20 29 31 6 13 37 38 3 35 27 34 12 30 33 1 4 28 25 24 22 26 19 17 15 14 8 5 11 2
40 1
0 0 0 1 3 4 6 6 6 8 10 11 11 11 12 15 16 17 16 17 20 19 22 21 23 23 25 27 28 29 30 31 32 31 34 34 34 36 38
24 5 2 39 38 9 21 36 33 7 3 37 35 32 14 20 26 34 18 17 31 13 30 28 27 29 23 22 25 16 19 15 11 12 10 8 6 1 4
100 1
0 0 1 2 2 5 5 5 5 9 7 11 9 10 13 14 13 14 15 19 17 21 19 20 22 23 25 25 27 29 29 30 32 33 32 32 33 36 37 37 39 41 39 42 43 45 43 45 45 47 49 48 52 52 52 55 54 56 57 56 57 61 60 63 62 64 64 66 68 69 70 68 71 72 73 74 76 77 78 77 77 81 81 81 83 83 85 86 86 86 88 91 89 90 93 95 93 97 96
67 26 34 18 99 46 31 38 75 84 24 94 90 4 35 80 44 20 96 12 11 42 19 23 65 51 95 73 15 16 53 40 82 36 28 98 79 78 50 6 8 59 3 41 47 14 71 7 92 70 69 97 49 91 1 62 10 93 61 89 57 58 87 54 88 85 83 81 77 86 76 74 68 72 64 66 60 56 55 63 52 48 43 45 39 37 33 32 30 29 27 22 25 17 13 21 9 5 2
1000 1
0 1 1 0 3 5 6 6 5 4 10 8 8 11 10 15 13 12 17 15 15 19 19 19 23 20 25 24 24 28 25 27 28 31 34 34 31 34 37 38 36 41 40 43 43 45 45 42 47 47 48 50 48 50 52 55 56 56 58 58 55 61 57 63 61 63 66 64 63 67 66 71 72 69 73 74 75 76 76 76 80 79 77 79 83 84 81 87 84 85 85 89 91 93 92 91 93 92 95 96 100 99 97 98 103 104 103 105 104 107 106 107 110 112 110 114 114 112 118 118 119 118 120 118 123 124 124 124 126 124 128 131 127 132 133 132 133 137 138 137 136 141 140 143 141 143 146 143 144 145 149 150 150 150 153 153 153 157 157 156 156 157 160 160 163 161 162 163 167 168 168 170 168 172 172 171 176 172 175 177 180 176 181 179 181 180 185 184 186 184 190 190 190 190 191 192 192 196 193 197 200 197 199 200 201 204 204 206 205 205 207 207 208 213 212 211 215 215 217 214 215 218 217 219 224 225 226 227 224 226 228 229 232 233 229 235 233 232 236 239 238 238 237 238 243 241 243 246 245 249 249 249 248 250 251 255 252 254 255 257 258 259 257 258 262 265 264 262 266 267 265 270 269 272 271 272 271 276 274 274 276 278 277 278 284 284 281 285 287 287 286 287 287 289 294 293 295 293 293 294 300 297 298 299 302 303 302 302 308 307 307 307 307 310 312 311 315 315 318 318 316 316 318 321 323 321 324 327 324 328 327 327 327 333 334 335 333 336 334 334 338 339 340 340 343 345 345 347 345 346 346 351 347 353 354 351 355 353 353 354 358 357 362 361 364 364 364 365 368 367 366 369 367 370 372 373 374 377 375 377 378 376 381 379 380 382 385 382 386 385 387 386 392 391 392 393 394 397 398 398 397 399 398 400 400 403 401 403 408 408 409 407 412 412 409 410 411 412 417 414 420 420 417 419 419 423 421 427 426 429 425 426 431 428 433 432 436 434 438 435 440 437 440 442 444 444 446 443 448 445 445 451 447 448 449 454 453 452 457 459 455 456 459 458 464 461 461 464 464 465 470 470 470 473 473 475 473 475 473 479 479 479 482 480 483 483 485 486 488 487 486 491 490 490 494 494 494 492 497 494 495 498 502 502 504 502 502 507 504 504 507 509 509 512 512 513 513 517 515 514 517 518 522 523 519 521 526 527 526 524 528 526 527 530 530 530 532 534 535 536 537 541 542 542 539 545 546 546 547 545 548 548 547 549 554 550 554 553 555 556 558 556 561 559 561 564 566 562 566 567 569 566 567 573 572 571 574 577 578 579 576 577 582 583 584 581 583 584 588 587 590 586 589 592 592 592 593 596 598 597 596 597 602 601 602 602 603 603 606 604 610 608 611 611 610 612 612 617 617 614 619 619 619 618 622 620 624 627 627 627 627 628 631 633 632 635 636 632 635 636 640 641 638 643 644 644 643 642 648 646 646 648 649 650 653 652 652 656 653 658 655 660 660 663 660 664 661 666 663 664 666 670 671 669 670 671 675 674 675 675 680 678 682 678 679 684 686 684 684 689 690 691 691 693 691 694 694 696 696 697 700 701 702 700 700 704 706 703 707 708 708 709 711 712 710 711 715 714 716 716 717 721 721 722 722 720 723 726 727 727 727 727 728 731 730 735 731 733 733 735 736 738 738 739 741 745 745 743 748 749 746 747 752 753 749 753 753 755 755 754 758 756 762 759 764 764 762 766 763 765 766 767 767 769 773 774 775 772 777 777 780 778 782 783 779 785 784 786 785 784 789 789 791 790 789 791 793 795 794 795 800 797 801 799 803 802 805 804 804 807 808 810 810 812 814 812 811 813 816 816 816 821 817 821 820 820 824 825 824 827 826 829 830 829 834 834 831 832 834 835 838 838 840 840 840 840 841 843 848 848 847 851 847 852 852 851 854 853 858 854 855 861 858 860 861 863 864 867 868 865 870 869 869 872 870 875 874 872 874 874 877 878 878 882 881 882 882 885 883 887 885 887 888 891 892 893 894 895 893 896 896 900 902 899 903 904 901 906 906 908 908 908 912 909 913 910 916 915 917 918 917 920 922 923 922 922 921 924 927 926 927 927 929 932 932 931 934 934 937 934 939 939 938 938 941 940 941 945 947 946 948 948 948 949 951 950 952 957 958 957 960 960 957 959 960 963 961 966 966 965 969 970 967 970 970 972 975 976 974 978 979 978 979 981 984 982 981 985 986 987 986 986 992 988 989 993 992 997 998
757 897 994 744 292 309 493 2 489 231 544 995 894 623 677 634 557 520 633 467 216 195 326 210 953 845 125 389 260 314 866 511 244 713 18 839 325 842 406 595 402 898 989 889 332 973 305 637 796 930 242 113 396 570 21 687 505 223 681 876 306 404 51 148 806 291 828 234 70 430 221 531 657 600 142 724 833 911 474 164 476 667 729 282 695 109 65 22 209 54 477 263 33 844 630 837 967 154 996 734 360 990 988 39 879 463 310 116 665 320 850 484 319 771 481 787 819 371 35 220 631 122 980 907 825 349 629 625 94 568 488 67 101 352 594 742 503 621 639 528 46 211 88 501 993 480 413 968 683 280 933 471 954 415 60 737 205 201 441 698 962 533 187 699 802 185 944 836 925 100 429 886 203 659 890 926 495 240 82 366 253 673 198 686 222 781 884 942 540 395 37 174 815 147 575 875 508 86 760 696 688 668 450 189 30 580 135 739 750 123 317 422 191 239 891 672 341 725 261 26 424 159 788 883 329 393 626 914 591 301 303 609 472 20 388 527 961 186 680 129 622 28 194 722 202 983 776 786 792 290 719 929 228 718 16 740 579 283 654 662 323 856 525 500 585 552 605 15 651 607 676 685 108 936 650 390 462 578 300 705 818 236 646 538 606 645 560 7 384 964 178 909 248 798 379 296 565 959 534 943 273 510 516 814 849 121 247 269 146 543 96 363 218 139 469 183 318 935 751 359 173 506 542 468 62 111 822 350 956 551 405 746 647 938 456 32 155 12 692 9 987 230 955 895 620 679 257 831 102 714 499 188 279 274 117 846 754 919 256 619 466 928 227 27 830 342 977 514 562 991 78 356 971 14 892 901 120 536 507 439 59 165 268 863 246 478 827 399 44 61 337 315 950 152 768 661 313 252 99 241 286 58 958 114 29 949 809 613 862 541 848 766 418 322 947 24 644 905 304 475 731 344 857 946 119 95 682 770 196 813 924 169 761 167 548 330 134 780 975 351 238 572 590 375 999 460 887 599 339 880 68 496 64 348 931 90 275 598 859 497 106 614 843 298 858 519 461 969 459 951 985 162 312 166 556 529 774 372 214 49 383 940 515 331 299 277 765 115 758 362 998 550 671 158 11 455 712 87 666 346 266 392 563 853 126 199 308 675 157 181 655 454 494 611 840 457 225 882 226 104 490 400 878 180 138 537 381 561 615 271 610 177 235 823 817 105 328 903 357 670 558 265 438 984 453 434 604 769 428 416 664 587 213 98 616 835 410 293 369 764 355 811 262 81 923 354 826 161 208 193 92 386 643 601 151 732 873 449 653 532 820 638 797 448 338 922 902 288 512 89 526 84 149 427 382 259 920 336 206 997 110 871 966 285 649 378 834 870 763 447 10 130 443 487 421 4 255 900 485 596 553 759 992 755 592 808 143 144 804 981 709 979 440 435 182 335 963 53 799 749 80 986 79 707 793 896 284 521 433 982 978 706 974 420 414 794 704 76 918 976 790 446 140 251 431 412 972 48 176 970 784 137 407 791 865 425 401 419 965 783 915 748 278 42 913 368 376 743 912 41 74 782 171 373 778 69 133 738 127 36 733 861 365 772 855 960 728 586 581 571 576 957 569 952 726 948 720 945 711 716 941 939 934 932 937 927 921 917 916 908 910 904 906 899 893 888 885 881 874 877 872 869 868 867 860 864 852 854 847 851 838 841 832 829 824 816 821 812 807 810 803 805 801 800 795 789 785 779 775 777 767 762 756 773 753 752 745 747 736 741 735 730 723 727 721 715 717 710 708 702 703 701 700 697 694 693 690 691 684 678 689 674 663 660 669 658 652 656 648 641 640 642 636 635 632 628 627 624 618 612 617 608 602 603 593 589 597 588 584 582 583 577 574 573 567 566 559 564 554 549 547 555 546 545 539 530 524 535 523 522 518 517 513 509 504 502 498 492 486 491 483 482 479 473 465 470 464 458 452 451 445 444 437 442 436 432 423 426 417 411 408 403 409 398 397 394 391 385 387 380 374 377 370 364 367 361 353 358 347 343 340 345 334 333 324 327 316 311 321 302 297 307 295 294 287 281 289 272 276 270 267 264 258 250 254 249 245 243 237 232 233 224 219 229 215 217 212 207 204 197 200 192 190 179 184 175 172 170 163 160 168 156 153 150 145 141 136 131 128 132 118 124 112 107 97 103 93 91 85 77 83 75 73 71 72 66 63 57 56 52 55 50 47 43 45 40 38 34 31 25 23 19 17 13 8 6 5 3 1
100 1
0 1 2 3 3 2 5 7 5 6 10 8 9 13 12 13 16 17 17 17 17 18 20 22 24 25 23 27 25 27 28 29 31 31 34 32 36 34 38 38 37 40 42 42 43 45 45 45 47 49 49 50 49 52 52 53 53 57 56 59 59 60 61 61 62 62 66 67 66 68 67 69 71 70 72 74 75 76 75 77 80 78 80 83 81 83 84 85 86 88 90 89 89 90 94 95 95 95 96
63 91 39 97 65 4 44 98 73 26 14 54 11 71 41 51 79 48 30 93 15 35 92 46 58 82 99 55 19 87 37 57 64 12 61 21 8 84 96 52 7 78 50 89 76 74 33 70 10 68 6 36 32 67 86 29 83 24 95 25 22 94 90 18 88 81 80 85 75 72 69 77 66 60 59 62 56 53 47 49 45 42 40 43 38 31 28 34 27 20 23 16 13 17 9 5 3 2 1
40 1
0 1 1 1 2 4 5 5 8 9 9 10 11 13 12 15 14 15 16 17 20 21 20 23 24 25 26 26 26 27 30 30 31 33 33 33 35 35 36
29 28 3 22 18 19 39 6 21 7 34 16 15 37 32 4 12 38 35 36 10 33 31 30 26 27 25 23 24 20 17 14 13 11 9 8 2 5 1
40 1
0 1 1 2 4 3 4 6 8 7 10 9 10 12 12 13 15 17 16 19 18 21 21 22 24 25 26 25 26 29 30 30 30 32 32 34 35 37 37
28 31 33 11 36 14 5 27 39 38 37 23 34 20 35 19 32 16 13 30 10 29 7 26 4 2 25 22 24 21 17 18 15 12 9 8 6 3 1
40 1
0 0 2 2 2 4 5 7 8 9 9 11 11 11 12 15 16 17 17 19 19 21 21 22 23 24 25 25 26 29 28 31 32 33 34 35 34 36 38
14 20 30 1 27 10 18 39 37 3 6 13 38 36 4 35 29 34 26 33 24 32 22 28 31 25 23 21 19 17 16 15 12 11 9 8 7 5 2
40 1
0 1 1 3 3 5 4 5 7 8 9 10 10 13 14 13 14 15 16 18 19 19 20 23 22 24 25 25 28 27 28 30 32 32 33 34 34 36 38
12 29 39 37 6 2 11 26 17 21 24 31 38 34 36 9 28 23 35 33 18 20 15 32 14 7 4 30 27 25 22 19 13 16 10 8 3 5 1
40 1
0 1 2 3 2 4 6 7 7 9 8 11 12 13 12 14 15 16 18 17 18 21 20 22 24 25 26 25 28 27 28 31 32 32 32 35 36 37 37
33 23 19 39 20 5 17 15 34 38 10 30 36 37 27 29 26 35 9 32 31 28 25 24 22 21 18 16 14 13 12 11 8 7 6 3 4 2 1
10 1
0 1 2 2 3 5 5 6 7
4 8 9 7 5 6 3 1 2
10 1
0 0 1 2 4 5 6 7 7
8 9 7 6 3 1 5 4 2
10 1
0 1 2 3 4 5 6 7 8
9 8 7 6 4 5 3 2 1
10 1
0 0 2 2 3 5 6 7 8
4 9 8 7 1 6 5 3 2
10 1
0 1 1 3 4 5 6 6 8
9 8 2 7 6 5 4 3 1
10 1
0 0 2 2 4 4 6 6 8
7 3 5 1 9 8 6 4 2
10 1
0 1 2 3 3 4 6 6 8
9 7 8 4 6 3 5 1 2
10 1
0 1 2 3 3 4 5 7 7
8 6 9 7 4 3 5 2 1
10 1
0 1 1 2 3 5 5 7 7
8 2 4 9 6 5 7 3 1
10 1
0 0 2 2 4 5 5 7 8
9 6 3 1 7 5 4 2 8
10 1
0 0 1 2 3 5 6 7 7
8 4 2 9 6 7 3 5 1
10 1
0 1 2 2 3 5 5 6 8
9 4 8 7 6 5 2 3 1
10 1
0 0 2 3 3 5 6 6 8
8 9 4 7 6 1 3 5 2
10 1
0 1 1 2 4 4 5 7 8
6 8 9 7 3 5 4 1 2
10 1
0 1 1 3 3 4 6 6 7
2 9 7 8 6 5 3 4 1
10 1
0 1 1 2 4 5 6 6 8
9 3 7 8 5 4 6 2 1
10 1
0 1 2 2 3 4 5 6 7
9 8 6 4 5 3 2 7 1
10 1
0 0 1 2 3 5 5 7 8
6 9 8 4 7 2 5 3 1
10 1
0 1 1 3 4 5 6 6 8
9 7 2 8 6 5 3 4 1
10 1
0 1 1 3 3 4 6 6 8
7 2 5 9 6 8 3 1 4
