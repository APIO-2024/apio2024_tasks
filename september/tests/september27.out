7ckgnn4wyi495puj3ibqf81dqvapyv6b
8
8
4
8
9
8
7
9
7
9
5
9
5
7
8
9
9
9
7
9
37
38
36
33
32
86
917
82
36
37
38
35
37
7
9
8
9
9
9
6
8
7
5
7
8
7
7
8
7
6
9
8
6
